src/main/java/org/folio/rest/impl/LoginAPI.java

```diff
@@ -313,14 +315,31 @@ private Response tokenResponse(JsonObject tokens) {
         .put(ACCESS_TOKEN_EXPIRATION, accessTokenExpiration)
         .put(REFRESH_TOKEN_EXPIRATION, refreshTokenExpiration)
         .toString();
+    var okapiPath = getPathFromOkapiUrl(okapiUrl);
     return Response.status(201)
-        .header(SET_COOKIE, accessTokenCookie(accessToken, accessTokenExpiration))
-        .header(SET_COOKIE, refreshTokenCookie(refreshToken, refreshTokenExpiration))
+        .header(SET_COOKIE, accessTokenCookie(accessToken, accessTokenExpiration, okapiPath))
+        .header(SET_COOKIE, refreshTokenCookie(refreshToken, refreshTokenExpiration, okapiPath))
         .type(MediaType.APPLICATION_JSON)
         .entity(body)
         .build();
   }
 
+  /**
+   * Return the path component of the okapiUrl without tailing '/',
+   * return empty string on parse error.
+   */
+  static String getPathFromOkapiUrl(String okapiUrl) {
+    try {
+      var path = new URL(okapiUrl).getPath();
+      if (path.endsWith("/")) {
+        return path.substring(0, path.length() - 1);
+      }
+      return path;
+    } catch (MalformedURLException e) {
+      return "";
+    }
```

Comment:
Shouldn't we at least log the `MalformedURLException` here to help people understand why the okapi url isn't appearing in their cookie? Or why catch the exception at all? Just let it fail.

Reply:
Ok I now see that it is just getting the path of the URL (not the whole URL) and using the `new Url` to get that, however I think the point still holds. Let's say that something is wrong with the url. Maybe it works in their system but doesn't parse with `new Url` for some reason. In that case they're silently going not get `okapi` in the path and their logins will fail without any indication of why it is failing. Why not log it or let it fail which will also log it with a stack trace (hopefully).

`getPath` returns an empty string if there is no path doesn't exist. This returns an empty string if something is wrong with the URL.